nextcord/application_command.py

```diff
@@ -1191,6 +1239,8 @@ def __init__(
         )
 
         self.autocomplete_callback: Optional[Callable] = autocomplete_callback
+        if issubclass(self.default, InteractionDefault):
```

Comment:
How does this even work? It's checking the current default on if it's a subclass but not checking the actual provided default

Reply:
Note, it also appears to override it on the line after regardless..?